embeddings/evaluator/sequence_labeling_evaluator.py

```diff
@@ -28,42 +30,41 @@ class TaggingScheme(str, Enum):
 
     def __init__(
         self,
+        return_input_data: bool = True,
         evaluation_mode: EvaluationMode = EvaluationMode.CONLL,
         tagging_scheme: Optional[TaggingScheme] = None,
     ) -> None:
-        super().__init__()
-        self.metric = self._get_metric(evaluation_mode, tagging_scheme)
+        super().__init__(return_input_data)
+        self.evaluation_mode = evaluation_mode
+        self.tagging_scheme = tagging_scheme
 
     def _get_metric(
         self,
-        evaluation_mode: EvaluationMode,
-        tagging_scheme: Optional[TaggingScheme] = None,
     ) -> Union[HuggingFaceMetric, UnitSeqevalMetric]:
```

Comment:
```suggestion
    ) -> HuggingFaceMetric:
```

`UnitSeqevalMetric` inherits from HuggingFaceMetric